README.md

```diff
@@ -29,16 +29,38 @@ The Delta Sharing Python Connector is a Python library that implements the [Delt
 
 ## System Requirements
 
-Python 3.6+
+Python 3.8+
```

Comment:
Shall we clarify with two bullets that, for version 1.1+ it's what you wrote. 
For version before 1.1, it's Python 3.6+?

Reply:
maturin doesnt work with anything less than python 3.8 right? So it would be 3.8 for everything if I am not mistaken

Reply:
discussed offline.